### PR TITLE
ZB Integration

### PR DESCRIPTION
**Pull request by estherleongym**
- Implemented Pairs, Tickers, OrderBook and Trades for ZB exchange
- Related to issue #276 
- [ X ] I have added Specs
- [ X ] (If implementing Market Ticker) I have verified that the `volume` refers to BASE
- [ X ] (If implementing Market Ticker) I have verified that the `base` and `target` is assigned correctly

### CODE REVIEW

File: lib/cryptoexchange/exchanges/zb/services/order_book.rb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+          order_book.bids      = adapt_orders(output["bids"])
+          order_book.timestamp = timestamp
+          order_book.payload   = output
+          order_book
+        end
+
+        def adapt_orders(orders)
+          orders.collect do |order_entry|
+            price, amount = order_entry
+            Cryptoexchange::Models::Order.new(price: order_entry[0],
```
Comment (Swingcloud) [inline lib/cryptoexchange/exchanges/zb/services/order_book.rb]: if price and amount were named, then you can just use `price` and `amount` here, maybe don't need to use `order_entry` here

File: lib/cryptoexchange/exchanges/zb/services/market.rb [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+          end
+        end
+        
+        def fetch(market_pair)
+          output = super(ticker_url(market_pair))
+          adapt(output, market_pair)
+        end
+
+        def ticker_url(market_pair)
+          "#{Cryptoexchange::Exchanges::Zb::Market::API_URL}/ticker/?market=#{market_pair.base.downcase}_#{market_pair.target.downcase}"
```
Comment (Swingcloud) [inline lib/cryptoexchange/exchanges/zb/services/market.rb]: seems ticker `/` this slash can remove

**Comment by estherleongym** [comment]
@Swingcloud OK, amended according to your comments :) thanks!

**Comment by tmlee** [comment]
Creating PR https://github.com/coingecko/cryptoexchange/pull/310 after conflict resolution

**Pull request closed without merging**